website/docs/advanced/verification-methods.md

```diff
@@ -23,23 +32,30 @@ const customVerifier = {
       reason: {
         code: 0,
         codeString: "SKIPPED",
-        message: `Document doesn't have version equal to 'https://schema.openattestation.com/2.0/schema.json'`
-      }
+        message: `Document doesn't have version equal to 'https://schema.openattestation.com/2.0/schema.json'`,
+      },
     };
   },
-  test: () => document.version === "https://schema.openattestation.com/2.0/schema.json"
-}
+  test: () => document.version === "https://schema.openattestation.com/2.0/schema.json",
+  // ... below is verify function
+};
 ```
 
 > we use `DOCUMENT_INTEGRITY` type because we check for the content of the document.
 
-### Document hold correct `name` property
-Once we have decided `when` the verification method run, it's time to write the logic of the verifier in the `verify`. We will use [getData](/docs/component/open-attestation#retrieving-document-data) utility to access the data of the document and return the appropriate fragment depending on the content:
+### Document holds correct `name` property
+
+Once we have decided `when` the verification method run, it's time to write the logic of the verifier in the `verify` method. We will use [getData](/docs/component/open-attestation#retrieving-document-data) utility to access the data of the document and return the appropriate fragment depending on the content:
 
 ```javascript
+// index.js
+const { verificationBuilder, openAttestationVerifiers, isValid } = require("@govtechsg/oa-verify");
+const { getData } = require("@govtechsg/open-attestation");
+const document = require("./document.json");
+
 const customVerifier = {
-  // ... above are skip and test function
-  verify: async document => {
+  // ... above are skip and test functions
```

Comment:
maybe, add this into the customVerifier that we created above ?
or create the property and add a comment inside saying content has been defined above.

Reply:
ok done, added the test and skip properties inside here which points to the previous section. should be clearer now instead of just showing comments.